### DISCUSSION TITLE
milvus http calls are much slower than grpc calls

### DISCUSSION THREAD

**Discussion by zhangchao** [milvus-io/milvus#25795] [Q&A and General discussion]
we did some test for milvus api , we found HTTP call is very slower than grpc , but I didn't know why ?  

HTTP :
<img width="1059" alt="image" src="https://github.com/milvus-io/milvus/assets/272933/3ffd4ae6-fd3e-4403-88bf-1f532ccd859e">

but for grpc only 80ms

**Comment by yhmo** [comment]
Possible reason: the consistency level is "Strong".

**Reply by zhangchao** [reply]
I don't think so because we use same query for HTTP and GRPC . I check the logic   http --> grpc proxy --> grpc handler .   not sure if message convert to PB is slow . we have only 96 float vector

**Comment by xiaofan-luan** [comment]
> I don't think so because we use same query for HTTP and GRPC . I check the logic http --> grpc proxy --> grpc handler . not sure if message convert to PB is slow . we have only 96 float vector

Milvus 2.2.12 released new high level Restful APIs. 
Could you test it and give some feedback on the new Restful APIs?
